packages/sdk/src/endpoints/IdentityMetadataEndpoint.ts

```diff
@@ -0,0 +1,16 @@
+import { ConnectorHttpResponse, ConnectorIdentityMetadata } from "../types";
+import { Endpoint } from "./Endpoint";
+
+export class IdentityMetadataEndpoint extends Endpoint {
+    public async upsertIdentityMetadata(request: any): Promise<ConnectorHttpResponse<ConnectorIdentityMetadata>> {
```

Comment:
This `any` should be `ConnectorIdentityMetadata`

Reply:
good catch